src/main/java/frc/robot/subsystems/indexer/IndexerIO.kt

```diff
@@ -0,0 +1,50 @@
+package frc.robot.subsystems.indexer
+
+import com.revrobotics.CANSparkMax
+import com.revrobotics.CANSparkMaxLowLevel
+import edu.wpi.first.math.geometry.Rotation2d
+import edu.wpi.first.wpilibj.DigitalInput
+import frc.robot.CANDevice
+import org.littletonrobotics.junction.LogTable
+import org.littletonrobotics.junction.inputs.LoggableInputs
+
+interface IndexerIO {
+    class IndexerInputs : LoggableInputs {
```

Comment:
By convention, this should be named just `Inputs` so it's referred to by `IndexerIO.Inputs`.